### PR TITLE
Territorial acknowledgement

### PR DESCRIPTION
**Pull request by arshiamufti**
I added this in policies because that's what we agreed on, but I thought about this some more and I think it makes more sense to add in in the "About Us" page. You can make arguments for why it doesn't belong in "About Us" too but I think there's a stronger case for it not being a policy--this is about us (the organizers) *acknowledging* something rather than *enforcing* it like a no-alcohol policy. Still, you could also say that this statement isn't "About *Us*". It's about aboriginal people. So another option is to put this in a separate page and link to it from our about page ("read our statement of acknowledgement of territory here") (but it wouldn't be as prominent).

Also, I took the text entirely from a WiCS issue that Evy was working on.

### CODE REVIEW

File: _pages/policies.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+## Territorial Acknowledgement ##
+
+StarCon would like to acknowledge that we are on the traditional territory of
+the Neutral, Anishnawbe and Haudenosaunee peoples. The University of Waterloo is
+situated on the Haldimand Tract, the land promised to the Six Nations that
+includes six miles on each side of the Grand River. It is important to remember
+that the land that we are using today for this event was stolen, and that there
+is lots of work to do to bring back the rights that indigenous and Native people
+deserve.
```
Comment (evykassirer) [inline _pages/policies.md]: I think we should add something about how  this relates to our mission to be inclusive of people from diverse backgrounds, and that in doing so it's important to recognize this kind of thing. Not totally sure how to word that

**Comment by arshiamufti** [comment]
i'll move this into it's own file and link to this from our home page (cc @evykassirer)

**Comment by evykassirer** [comment]
It's pretty short, why does it need its own file? People are much more likely to read it if it's just on the homepage

**Comment by arshiamufti** [comment]
I was thinking that later on we'll have to add sponsors, keynote speakers etc to the home page. Also, since you suggested that we add a couple of more sentences, the statement is going to be longer than it is right now. We don't want to make the home page too long.

**Comment by evykassirer** [comment]
Can we leave it there for now and move it if it gets too long later?

**Comment by arshiamufti** [comment]
Sure thing! I'll update this today evening.

**Pull request merged**